src/formats/palette.h

```diff
@@ -22,6 +22,28 @@
 extern "C" {
 #endif
 
+// All sprites have holes, these holes need to be transparent.
+#define SPRITE_TRANSPARENT_INDEX 0
+
+// Backgrounds have no transparent pixels and need all pixels to be rendered.
+#define BACKGROUND_TRANSPARENT_INDEX -1
+
+// Force an opaque menu for now.
+#define MENU_TRANSPARENT_INDEX -1
+
+// Portraits have a transparancy that needs to offset past other items in the global palette.
+#define PORTRAIT_TRANSPARENT_INDEX 0xD0
```

Comment:
Hmmh. These specific ones should probably be in the files where they are used. They are not generic, after all.

Reply:
It was like that originally and it was very painful to find all of them. I would really like them to be in 1 spot.